pkg/resources/services/service.go

```diff
@@ -71,7 +74,7 @@ func NewClusterIPForMinIO(t *miniov2.Tenant) *corev1.Service {
 		},
 	}
 	// check if the service is meant to be exposed
-	if t.Spec.ExposeServices != nil && t.Spec.ExposeServices.MinIO {
+	if t.Spec.ExposeServices == nil || t.Spec.ExposeServices.MinIO {
```

Comment:
this would make it so the default behavior is `LoadBalancer` for the service which I don't think is a good idea, we should keep the default as simple ClusterLoadbalancer, not every k8s cluster can expose the service as loadblanacer

Reply:
agreed, I'm just following the docs that says that this is the default :D 

But I agree with you, will revert this and probably we need to fix the doc?

Reply:
@dvaldivia fixed on the API docs. I don't know if I should run the make regen-crd-docs so I didn't for now (there are a lot of differences)